buildroot/arm.config

```diff
@@ -1,13 +1,10 @@
 BR2_arm=y
 BR2_CCACHE=y
-BR2_OPTIMIZE_3=y
 BR2_TOOLCHAIN_EXTERNAL=y
 BR2_TOOLCHAIN_EXTERNAL_BOOTLIN=y
 BR2_TOOLCHAIN_EXTERNAL_BOOTLIN_ARMV5_EABI_UCLIBC_STABLE=y
 BR2_TARGET_GENERIC_ROOT_PASSWD="root"
 BR2_TARGET_GENERIC_GETTY_PORT="ttyAMA0"
 BR2_ROOTFS_OVERLAY="../overlay-reboot"
-BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
-BR2_PACKAGE_BASH=y
```

Comment:
What shell do we boot to with `-i`? I assume we still have one, it's just whatever busybox has which I don't think is Bash?

Reply:
I use this with `-i` a lot, so I'm just checking this doesn't break that.

Reply:
I believe the shell is busybox ash.